homeassistant/components/netatmo/sensor.py

```diff
@@ -282,124 +282,99 @@ def __init__(self, data_handler, data_class_name, module_info, sensor_type):
                 f"{module_info.get('module_name', device['type'])}"
             )
 
-        self._name = (
+        self._attr_name = (
             f"{MANUFACTURER} {self._device_name} {SENSOR_TYPES[sensor_type][0]}"
         )
         self.type = sensor_type
-        self._state = None
-        self._device_class = SENSOR_TYPES[self.type][3]
-        self._icon = SENSOR_TYPES[self.type][2]
-        self._unit_of_measurement = SENSOR_TYPES[self.type][1]
+        self._attr_state = None
```

Comment:
```suggestion
```
I think self._attr_state = None is the default, so no need to initiate self._attr_state